src/migration.js

```diff
@@ -143,13 +144,13 @@ Migration.prototype.migrate = function(doneCb) {
                             step.status = statuses.skipped;
                             cb();
                         }else if(step.status === statuses.pending){
-                            step.up(db, function(err){
+                            step.up(client, function(err){
```

Comment:
should we be passing the client here? Based on existing migrations https://github.com/A24Group/chatster/blob/master/migrations/migration-1/0011-create-collection-Workflow.js it looks like we need to use the same object that is used here https://github.com/A24Group/mongration/pull/1/files#diff-6a95d95dc11285029c6031298106697f9fb7d43481b17f5cb038b362fbd49447R153